skiko/src/awtMain/kotlin/org/jetbrains/skiko/context/MetalContextHandler.kt

```diff
@@ -55,12 +64,31 @@ internal class MetalContextHandler(layer: SkiaLayer) : JvmContextHandler(layer)
     override fun flush() {
         super.flush()
         surface?.flushAndSubmit()
-        metalRedrawer.finishFrame()
+        finishFrame()
     }
 
     override fun rendererInfo(): String {
         return super.rendererInfo() +
-            "Video card: ${metalRedrawer.adapterName}\n" +
-            "Total VRAM: ${metalRedrawer.adapterMemorySize / 1024 / 1024} MB\n"
+                if (adapterInfo != null) {
+                    "Video card: ${adapterInfo.adapterName}\n" + "Total VRAM: ${adapterInfo.adapterMemorySize / 1024 / 1024} MB\n"
```

Comment:
Let's override `adapterInfo.toString()` instead. (or revert line break after +)

Reply:
Not sure that it is ok to have toString with \n. Break after + will cause auto-formatting to format it in a weird way.